tests/packagedcode/data/plugin/podspec-package-expected.json

```diff
@@ -0,0 +1,72 @@
+{
+  "headers": [
+    {
+      "tool_name": "scancode-toolkit",
+      "options": {
+        "input": "<path>",
+        "--json": "<file>",
+        "--package": true,
+        "--processes": "-1",
+        "--strip-root": true
+      },
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "message": null,
+      "errors": [],
+      "extra_data": {
+        "files_count": 1
+      }
+    }
+  ],
+  "files": [
+    {
+      "path": "SwiftLib.podspec",
+      "type": "file",
+      "packages": [
+        {
+          "type": "podspec",
+          "namespace": null,
+          "name": "SwiftLib",
+          "version": "0.0.1",
+          "qualifiers": {},
+          "subpath": null,
+          "primary_language": "Swift",
+          "description": " This CocoaPods library helps you perform calculation.",
+          "release_date": null,
+          "parties": [
+            {
+              "type": null,
+              "role": "author",
+              "name": "alizainprasla",
+              "email": "alizainprasla@gmail.com",
+              "url": null
+            }
+          ],
+          "keywords": [],
+          "homepage_url": null,
+          "download_url": null,
+          "size": null,
+          "sha1": null,
+          "md5": null,
+          "sha256": null,
+          "sha512": null,
+          "bug_tracking_url": null,
+          "code_view_url": null,
+          "vcs_url": null,
+          "copyright": null,
+          "license_expression": "mit",
+          "declared_license": "MIT",
```

Comment:
The original license was not MIT but:
`s.license          = { :type => 'MIT', :file => 'LICENSE' }`
You need to keep the original data structure so instead of:
`"declared_license": "MIT",`
if should be:
`"declared_license": {"type": "MIT", "file": "LICENSE"}`
And the license detection applied to that should be updated accordingly and overriden specially for Cocoapods

Reply:
Fixed:)